spec/lib/reports/orders_and_distributors_report_spec.rb

```diff
@@ -151,7 +151,6 @@
         subject # build context first
 
         expect { subject.table_rows }.to query_database [
-          "Enterprise Pluck",
```

Comment:
Ah ha, there is less queries, yay! It would be nice to see these with the commit that caused it, so we can see exactly what change caused it.